docs/kf.dev/content/en/docs/developers/reference-apps/link-shortener/_index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Link Shortener App "
+linkTitle: "Link Shortener App"
+weight: 20
+description: >
+  Learn how to deploy the Link Shortener reference application originally built
+  for Cloud Foundry and the Google Cloud service broker.
+---
+
+This example Spring application is an enterprise-ready link-shortener.
+The URL shortener has the following behaviour:
+
+ * URLs internal to your own domain get automatically redirected.
+ * URLs on the Internet are scanned for vulnerabilities:
+   * If the URL is malicious, the redirect is blocked.
+   * If the URL is benign, the user is shown a preview of the site and clicking
+through opens in a new tab to protect privacy.
+   * If the vulnerability scanner is down, a warning is shown.
+
+{{% alert title="Costs Money" color="warning" %}}
+Following these instructions will result in creation of a CloudSQL database instance
+that will incur billing charges on your GCP account.
+{{% /alert %}}
+
+## Prerequisites
+
+[google-broker]: /docs/operators/service-brokers/google-cloud/
+1. A Kf cluster and the `kf` CLI installed on your workstation
```

Comment:
Can we link to install instructions for kf?

Reply:
Done